addons/api/mirage/config.js

```diff
@@ -18,11 +18,6 @@ export default function () {
   // delay for each request, automatically set to 0 during testing
   this.timing = 1;
 
-  // Allow any configured API host to passthrough, which is useful in
-  // development for testing against a locally running backend.
-  if (config.api.host) this.passthrough(`${config.api.host}/**`);
-  this.passthrough();
```

Comment:
Was there a reason we needed this `passthrough`? I don't understand the reasoning why we would want to have requests passthrough to our current domain, i.e. localhost?

I would actually expect if we're hitting an endpoint that's not being mocked, that it _should_ automatically fail.